src/lib.rs

```diff
@@ -192,4 +197,12 @@ mod tests {
         // `len` is a `String` method.
         assert!(nes.len() > 0);
     }
+
+    #[test]
+    fn format_test() {
+        let str = NonEmptyString::new("string".to_owned()).unwrap();
+        println!("{}", &str);
+
+        let _str: String = str.to_string();
```

Comment:
Could you please add an assertion on the result of `to_string()`, that way we solidify the implementation?